dangerfile.js

```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { danger, fail } from 'danger';
```

Comment:
I need to add the danger key to the travis-ci job for this to work

Reply:
Can we just use the built in git hub bot for this?

Reply:
yeah.... that make sense

Reply:
We need to add a `.github/changelog.yml` something like: https://github.com/cerner/terra-aggregate-translations/blob/master/.github/changelog.yml

Then we can remove danger and this file